doc/classes/ColorPicker.xml

```diff
@@ -186,6 +186,9 @@
 		<theme_item name="sample_bg" data_type="icon" type="Texture2D">
 			Background panel for the color preview box (visible when the color is translucent).
 		</theme_item>
+		<theme_item name="sample_revert" data_type="icon" type="Texture2D">
+			The icon for the revert button (visible on the middle of the "old" color when it differs from the currently selected color).
```

Comment:
Since the icon gets modulated, maybe it's worth mentioning that it should be bright.